.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
would rather not adopt editorconfig right now as we don't use it in any of our other products

Reply:
Removed, althought it's kinda sad. I think the inventor of editorconfig deserves a Nobel-prize. :D It essentially resolves the tabs vs spaces conflict because nobody has to care anymore.